mlrun/projects/project.py

```diff
@@ -4063,6 +4063,12 @@ def store_alert_config(
         db = mlrun.db.get_run_db(secrets=self._secrets)
         if alert_name is None:
             alert_name = alert_data.name
+        if alert_data.project is not None and alert_data.project != self.metadata.name:
+            logger.warn(
+                "Project in alert does not match project in operation",
+                project=alert_data.project,
+            )
```

Comment:
actually I think we should just overwrite the project. same as we do if you import a function from a different project